### PR TITLE
Frequentist - Refactor and clean up prior to UI changes

### PR DESCRIPTION
**Pull request by bttf**
This PR includes simple refactoring, type definition updates, and TODO placement in effort to prep for incoming UX changes for the Frequentist stats engine

No changes functionality-wise is introduced in this PR - it's primarily clean up and prep only.

### CODE REVIEW

File: packages/front-end/hooks/usePValueThreshold.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import useOrgSettings from "./useOrgSettings";
+
+export default function usePValueThreshold() {
+  const settings = useOrgSettings();
+  return settings.pValueThreshold || 0.05;
+}
```
Comment (bttf) [inline packages/front-end/hooks/usePValueThreshold.ts:6]: Default to `0.05` for now

File: packages/front-end/components/Metrics/MetricForm/index.tsx [hunk 2]
```diff
@@ -892,104 +897,14 @@ const MetricForm: FC<MetricFormProps> = ({
...
-                green, yellow, or red.
-              </small>
-            </div>
+            {!isFrequentist && (
+              <RiskThresholds
+                winRisk={value.winRisk}
+                loseRisk={value.loseRisk}
+                winRiskRegisterField={form.register("winRisk")}
+                loseRiskRegisterField={form.register("loseRisk")}
+              />
```
Comment (bttf) [inline packages/front-end/components/Metrics/MetricForm/index.tsx:906]: Simple refactor - just moved component into its own file, no changes

File: packages/front-end/components/Metrics/MetricForm/RiskThresholds.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import { FC } from "react";
+import { UseFormRegisterReturn } from "react-hook-form";
+
+const RiskThresholds: FC<{
+  winRisk: number;
+  winRiskRegisterField: UseFormRegisterReturn;
+  loseRisk: number;
+  loseRiskRegisterField: UseFormRegisterReturn;
+}> = ({ winRisk, winRiskRegisterField, loseRisk, loseRiskRegisterField }) => {
+  const riskError =
```
Comment (bryce-fitzsimons) [inline packages/front-end/components/Metrics/MetricForm/RiskThresholds.tsx:10]: We have an identical calculation happening in the parent index.tsx. We probably don't need to set the riskError string within the parent anymore, just use a boolean for blocking the modal's Submit.

**Review by bryce-fitzsimons — CHANGES_REQUESTED** [review]

File: packages/front-end/components/Metrics/MetricForm/RiskThresholds.tsx (hunk 3 above)
Comment (bttf) [reply packages/front-end/components/Metrics/MetricForm/RiskThresholds.tsx]: Ah yeah, good point. Updated

**Comment by bttf** [comment]
@bryce-fitzsimons Thanks, updated

**Review by bryce-fitzsimons — APPROVED** [review]

**Review by bryce-fitzsimons — APPROVED** [review]

**Pull request merged**